test/v2/services/RetrieveUkPropertyAdjustmentsServiceSpec.scala

```diff
@@ -32,7 +32,7 @@ import scala.concurrent.Future
 class RetrieveUkPropertyAdjustmentsServiceSpec extends UnitSpec {
 
   private val nino = Nino("AA123456A")
-  private val correlationId = "X-123"
+  private implicit val correlationId = "X-123"
```

Comment:
Please remove once you have added the **implicit val correlationId: String = "X-123"** to the ServiceSpec and this class extends it instead (Change from UnitSpec to ServiceSpec)